src/libraries/System.Private.CoreLib/src/System/DateOnly.cs

```diff
@@ -82,26 +82,33 @@ public static DateOnly FromDayNumber(int dayNumber)
 
         /// <summary>
         /// Gets the year component of the date represented by this instance.
+        /// The returned value is an integer between 1 and 9999.
```

Comment:
This is not the correct documentation used for `DateTime`. The IntelliSense doc is located at https://github.com/dotnet/dotnet-api-docs . Xml doc in this repo is only used as a seed and not backported for changes. The actual documentation is located at https://github.com/dotnet/dotnet-api-docs/blob/e50aa1e11da935331d6eed7feb18853c8439b4f7/xml/System/DateTime.xml#L10471 . The description of property return value should be in the `<value>` tag.